maven/bnd-indexer-maven-plugin/src/main/java/aQute/bnd/maven/indexer/plugin/LocalIndexerMojo.java

```diff
@@ -26,14 +26,23 @@
 public class LocalIndexerMojo extends AbstractMojo {
 	private static final Logger	logger		= LoggerFactory.getLogger(LocalIndexerMojo.class);
 
+	private static final String ABS_URL_IDENTIFIER	= "///";
+
 	@Parameter(property = "bnd.indexer.input.dir", required = true)
 	private File				inputDir;
 
 	@Parameter(property = "bnd.indexer.output.file", defaultValue = "${project.build.directory}/index.xml")
 	private File				outputFile;
 
+	/**
+	 * This configuration parameter is used to relativize the path of the index
+	 * URIs stored in the index. By default these relative paths use the index
+	 * output location as their base directory, but a different base directory
+	 * can be supplied if needed.
+	 * The special string "///" can be used to disable the relativize process.
+	 */
 	@Parameter(property = "bnd.indexer.base.file")
-	private File				baseFile;
+	private String				baseFile;
```

Comment:
I think it would be better to have a new parameter `boolean relativize` who's default is `true` to match the existing default. Then we don't need a special marker value.

Reply:
Okay for me.
I was not sure, which is the preferred approach.